sdv/multi_table/base.py

```diff
@@ -82,3 +83,74 @@ def update_table_parameters(self, table_name, table_parameters):
     def get_metadata(self):
         """Return the ``MultiTableMetadata`` for this synthesizer."""
         return self.metadata
+
+    def _validate_foreign_keys(self, data):
+        errors = []
+        for relation in self.metadata._relationships:
+            child_table = data[relation['child_table_name']]
+            parent_table = data[relation['parent_table_name']]
+            child_row = child_table[relation['child_foreign_key']]
+            parent_row = parent_table[relation['parent_primary_key']]
+            missing_values = child_row[~child_row.isin(parent_row)].unique()
+            if any(missing_values):
+                message = ', '.join(missing_values[:5].astype(str))
+                if len(missing_values) > 5:
+                    message = f'({message}, + more).'
+                else:
+                    message = f'({message}).'
+
+                errors.append(
+                    f"Error: foreign key column '{relation['child_foreign_key']}' contains "
+                    f'unknown references: {message}'
```

Comment:
This message is missing the second half of the error: `All the values in this column must reference a primary key.`